doc/source/dev/roadmap-detailed.rst

```diff
@@ -401,8 +408,6 @@ The following improvements will help SciPy better serve this role.
 
 - Add fundamental and widely used hypothesis tests:
 
-  - Alexander-Govern test
-  - Somers' D
   - Kendall's tau-c
   - Page's L-test
```

Comment:
```suggestion
```
These are done. Many other SciPy stats things will be done very soon. E.g. Tukey-Kramer test is dependent on gh-13732 but otherwise is basically ready for a PR. I can take care of that.

Reply:
We could leave this issue open until the release.